java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -76,6 +76,24 @@ public interface SetBaseCommands {
      */
     CompletableFuture<Long> scard(String key);
 
+    /**
+     * Moves a set element from one set to another. Creates a new destination set if needed.<br>
```

Comment:
Moves `member` from the set at `source` to the set at `destination`, `removing` it from the source set. ...